### ISSUE TITLE
Code format - Suggestion

### ISSUE THREAD

**Issue by yuvrajsab** [csdu/sankalan-portal#20]
The script tag in layouts/app should move from body to head because it has defer attribute. which currently makes no sense.

*from*
```
<body>
    ...
    <script src="{{ mix('js/app.js') }}" defer></script>
</body>
```
*to*
```
<head>
    ...
    <script src="{{ mix('js/app.js') }}" defer></script>
</head>
```

**Comment by jatin69** [comment]
There will a lot of these issues in the codebase. It was developed under time constraints. A better way would be collective pull requests fixing lots of these at a time. Then @rumansaleem can review them and merge the changes.

**Comment by rumansaleem** [comment]
@yuvrajsab you can send a PR referencing this issue. I will review and merge them

**Comment by yuvrajsab** [comment]
sent

**Comment by jatin69** [comment]
@yuvrajsab Issues are usually closed by the reviewers who merge your pull request. They reference this issue in the merge commit, so we can track which issue was fixed by which pull request.

**Comment by rumansaleem** [comment]
resolved by #21
